src/main/java/com/resource/management/services/controller/DeleteAllServicesController.java

```diff
@@ -0,0 +1,29 @@
+package com.resource.management.services.controller;
+
+import com.resource.management.api.services.DeleteAllServicesRequest;
+import com.resource.management.api.services.ServicesListUpdatedNotification;
+import com.resource.management.services.model.Service;
+import com.resource.management.services.model.ServiceRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.stereotype.Controller;
+
+import java.time.Instant;
+import java.util.Comparator;
+import java.util.List;
+
+@Controller
+public class DeleteAllServicesController {
+    @Autowired
+    private ServiceRepository repository;
+
+    @MessageMapping("/deleteAllServices")
+    @SendTo("/topic/services")
+    public ServicesListUpdatedNotification handle(final DeleteAllServicesRequest request) {
+        repository.deleteAll();
+        List<Service> services = repository.findAll();
```

Comment:
I think that if you're deleting all the services than findAll() will return an empty list, so after deletion you can send a ServiceListUpdatedNotification with an empy list

Reply:
true...